runtime/onert/api/python/src/wrapper/nnfw_api_wrapper.cc

```diff
@@ -31,20 +31,21 @@ void ensure_status(NNFW_STATUS status)
     case NNFW_STATUS::NNFW_STATUS_NO_ERROR:
       return;
     case NNFW_STATUS::NNFW_STATUS_ERROR:
-      throw NnfwError("NNFW_STATUS_ERROR");
+      throw NnfwError("Error");
```

Comment:
Not sure if the changes done to the strings here bring that much value but changing "status error" to "error" in particular removes some information that could be valuable. The "status" word might be useful when receiving this message to locate the problem.

Reply:
OK, I can revert that. Strings update (if desired) can be done in a separate PR.